tsdat/io/readers/tar_reader.py

```diff
@@ -62,6 +62,7 @@ class Parameters(BaseModel, extra=Extra.forbid):
 
     parameters: Parameters = Parameters()
 
+    # TODO: The `input_key` parameter is not referenced in the docstring.
```

Comment:
good catch. it used to be called `file`, but it looks like I forgot to update the docstring when we made that change

Reply:
Resolved in 35470580f53a6a9bce0b79f199faebf27d01c408.